data/templates/ipsec/swanctl/remote_access.j2

```diff
@@ -32,6 +32,9 @@
             auth = {{ rw_conf.authentication.client_mode }}
 {% if rw_conf.authentication.client_mode.startswith("eap") %}
             eap_id = %any
+{% endif %}
+{% if rw_conf.authentication.client_mode == 'eap_tls' or rw_conf.authentication.client_mode == 'x509' %}
```

Comment:
You should use `eap-tls` over `eap_tls`. Please rephrase it to:

`{% if rw_conf.authentication.client_mode is vyos_defined('eap-tls') or rw_conf.authentication.client_mode is vyos_defined('x509') %}`

Else the code path is not hit for eap-tls client auth.

Reply:
Please also update the IPSec remote-access smoketest (test_07) in https://github.com/vyos/vyos-1x/blob/current/smoketest/scripts/cli/test_vpn_ipsec.py#L551 to validate this option.